platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/validation/DefaultInternalEventValidator.java

```diff
@@ -173,12 +173,20 @@ public DefaultInternalEventValidator(
     private boolean areRequiredFieldsNonNull(@NonNull final PlatformEvent event) {
         final GossipEvent gossipEvent = event.getGossipEvent();
         final EventCore eventCore = gossipEvent.eventCore();
-        if (eventCore == null
-                || eventCore.timeCreated() == null
-                || eventCore.version() == null
-                || eventCore.parents().stream().anyMatch(Objects::isNull)
-                || gossipEvent.eventTransaction().stream().anyMatch(DefaultInternalEventValidator::isTransactionNull)) {
-            nullFieldLogger.error(EXCEPTION.getMarker(), "Event has null field {}", gossipEvent);
+        String nullField = null;
+        if (eventCore == null) {
+            nullField = "eventCore";
+        } else if (eventCore.timeCreated() == null) {
+            nullField = "timeCreated";
+        } else if (eventCore.version() == null) {
+            nullField = "version";
+        } else if (eventCore.parents().stream().anyMatch(Objects::isNull)) {
+            nullField = "parent";
```

Comment:
code coverage indicates line 184 was missed.

Reply:
you must be looking at the old report, i added coverage for this